src/components/ModalPayment/SquareModal.tsx

```diff
@@ -61,117 +56,113 @@ const ModalPayment = ({
     return makeSquarePayment( nonce, payment, buyer )
       .then((res) => {
         if (res.status === 200) {
-          setIsShown(true)
-        } else {
-          setIsShown(false)
+          dispatch({type: SET_MODAL_VIEW, payload: 2})
+        } 
+      })
+      .catch((err) => {
+        if( err.response
```

Comment:
this is pretty gnarly — do you mind running yarn format && yarn eslint --fix before you merge?